tools/csi-deps.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh
+
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+
+DEST=/dest
+
+copy_deps() {
+    PROG="$1"
+
+    mkdir -p "${DEST}$(dirname $PROG)"
+
+    if [ -d "${PROG}" ]; then
+        rsync -av "${PROG}/" "${DEST}${PROG}/"
+    else
+        cp -Lv "$PROG" "${DEST}${PROG}"
+    fi
+
+    if [ -x ${PROG} -o $(/usr/bin/ldd "$PROG" >/dev/null) ]; then
+        DEPS="$(/usr/bin/ldd "$PROG" | /bin/grep '=>' | /usr/bin/awk '{ print $3 }')"
+
+        for d in $DEPS; do
+            mkdir -p "${DEST}$(dirname $d)"
+            cp -Lv "$d" "${DEST}${d}"
+        done
+    fi
+}
+
+# k8s.io/mount-utils
```

Comment:
Have you copied this from somewhere by any chance? Exactly how does the list below relate to mount-utils? It certainly looks like it does, but I wonder if you could add a comment explaining exactly how you came by it, which would make it easier to keep up to date in the future.